src/osha/oira/content/browser/survey_publication_count.py

```diff
@@ -0,0 +1,60 @@
+from io import StringIO
+from plone import api
+from Products.Five import BrowserView
+
+import csv
+
+
+fieldnames = ["country", "path", "count"]
+
+
+def get_survey_details():
```

Comment:
I strongly suggest to not define utility functions outside the view.
It will make harder to customize and reuse the code.
This should become a row property in the `SurveyPublicationCount` view.